modules/ROOT/pages/optimization.adoc

```diff
@@ -0,0 +1,35 @@
+[[optimization]]
+= Optimization
+
+
+This chapter contains some optimizations to improve the performance of a production system using `@neo4j/graphql`.
+
+== Schema Optimisations
```

Comment:
Will there be more optimization sections on this page in the future?

Reply:
Yes, the plan is to add a few things that we are planning, for "Schema" and "Queries"